src/elements/OcTextInput.vue

```diff
@@ -4,7 +4,7 @@
     :type="type"
     :value="value"
     :placeholder="placeholder"
-    :aria-label="placeholder"
+    :aria-label="label"
```

Comment:
Let's move here pls also docs about why is this different as in text area component.

Reply:
In what regard different? both have the value of placeholder mapped to the placeholder attribute; and the value of label to the aria-label attribute.

Reply:
The docs about the placeholder vs. label. Why the placeholder shouldn't be used as a label. "The attributes `placeholder` and `aria-label` have different functions..." As in text area.

Reply:
Got it. Added that bit.

Reply:
Thanks!